dspace-spring-rest/src/test/java/org/dspace/app/rest/CommunityRestRepositoryIT.java

```diff
@@ -253,17 +252,23 @@ public void findAllSubCommunities() throws Exception{
                 .withName("Sub Community")
                 .build();
 
-        Community child12 = CommunityBuilder.createSubCommunity(context, child1)
+        Community child12 = CommunityBuilder.createSubCommunity(context, parentCommunity)
```

Comment:
@Raf-atmire The naming suggests that this Community is a sub-community of `child1`, but it's not. Can you rename this variable to `child2`?